content/work/coachhub.md

```diff
@@ -146,8 +147,8 @@ easily access one another in dynamic and effective ways.
 
 {{ embed.icon_block(planning, 'Planning & Discovery Phase') }}
 
-[ORCAS]: http://orcasinc.com
-[MoodHacker]: http://www.orcasinc.com/products/moodhacker/
+[orcas]: http://orcasinc.com
```

Comment:
I didn't do this but it is showing as a change... I assume a formatting plugin did this